Tests/RevenueCatUITests/CustomerCenter/ManageSubscriptionsViewModelTests.swift

```diff
@@ -112,27 +327,36 @@ class ManageSubscriptionsViewModelTests: TestCase {
         expect(viewModel.state) == .error(error)
     }
 
+    private func reformat(ISO8601Date: String) -> String {
+        let formatter = DateFormatter()
+        formatter.dateStyle = .medium
+        return formatter.string(from: ISO8601DateFormatter().date(from: ISO8601Date)!)
+    }
+
 }
 
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 final class MockManageSubscriptionsPurchases: ManageSubscriptionsPurchaseType {
-
```

Comment:
This empty line should be here to match our SwiftStyleGuide https://github.com/RevenueCat/purchases-ios/blob/dbcfe0ceb55879c98d7069c2c788af90b76b27d7/Contributing/SwiftStyleGuide.swift#L18

Reply:
Good to know, thanks! https://github.com/RevenueCat/purchases-ios/pull/4046/commits/89d4a8d61b50b0a04c8ff82c97b3d8d7b9d5526d